src/i18n/nb_NO.ts

```diff
@@ -0,0 +1,861 @@
+        },
+        7: {
+            caption: 'CBRNe enhet',
+            color: '#770000',
+            coins: 25,
+            credits: 19_200,
+            minPersonnel: 1,
+            maxPersonnel: 6,
+            special: 'Nødvendig etter at du har bygd 11 brannstasjoner.',
+            schooling: 'Krever spesialistutdanning (CBRNe-enhet)',
+            shownSchooling: 'CBRNe-enhet',
+        },
+        8: {
+            caption: 'Patruljebil',
+            color: '#8b1818',
+            coins: 25,
+            credits: 5_000,
+            minPersonnel: 1,
+            maxPersonnel: 2,
+        },
+        9: {
+            caption: 'Luftambulanse',
+            color: '#e61919',
+            coins: 30,
+            credits: 300_000,
+            minPersonnel: 1,
+            maxPersonnel: 1,
+        },
+        10: {
+            caption: 'Snorkelbil',
+            color: '#dc1818',
+            coins: 25,
+            credits: 19_000,
+            minPersonnel: 1,
+            maxPersonnel: 6,
+            special:
+                'To purchase with credits it requires the rank: Captain, <br>Lower ranked members can purchase the vehicle for 25 Coins. <br>Quint acts as a Platform Truck and a Fire Truck.',
+        },
+        11: {
+            caption: 'Politihelikopter',
+            color: '#ca1616',
+            coins: 30,
+            credits: 300_000,
+            minPersonnel: 1,
+            maxPersonnel: 2,
+            schooling: 'Police - Police Aviation',
+            shownSchooling: 'Police Aviation',
+        },
+        12: {
+            caption: 'Pansret kjøretøy',
+            color: '#a51212',
+            coins: 25,
+            credits: 17_300,
+            minPersonnel: 6,
+            maxPersonnel: 6,
+            schooling: 'Police - Våpentrening',
+            shownSchooling: 'SWAT',
+            special: 'Nødvendig etter at du har bygt 8 politistasjoner.',
+        },
+        13: {
+            caption: 'Hundepatrulje',
+            color: '#aa2222',
+            coins: 25,
+            credits: 7_000,
+            minPersonnel: 1,
+            maxPersonnel: 2,
+            schooling: 'Politi - Hundepatrulje',
+            shownSchooling: 'Hundepatrulje',
+            special: 'Nødvendig etter at du har bygt 6 politistasjoner.',
+        },
+        14: {
+            caption: 'Politimotorsykkel',
+            color: '#662222',
+            coins: 18,
+            credits: 2_500,
+            minPersonnel: 1,
+            maxPersonnel: 1,
+            schooling: 'Politi - Politimotorsykkel',
+            shownSchooling: 'Politimotorsykkel',
+        },
+        15: {
+            caption: 'Delta kjøretøy',
+            color: '#332222',
+            coins: 23,
+            credits: 7_000,
+            minPersonnel: 2,
+            maxPersonnel: 4,
+            special: 'Required once you have built 8 police stations',
+            schooling: 'Police - SWAT',
+            shownSchooling: 'SWAT',
+        16: {
```

Comment:
Please add a row between row 235 and 236 with **},**

Reply:
ahh thx, i dont find the bug :D